scripts/travis-emscripten/build_emscripten.sh

```diff
@@ -54,14 +54,6 @@ fi
 
 WORKSPACE=/root/project
 
-# Increase nodejs stack size
```

Comment:
This was needed for compilation and for the resulting binary. Is it safe to remove it now?

Reply:
Yep, not needed anymore.

Reply:
Sorry, updated the comment, it was **not** needed for the binary, but the compilation process :)

Reply:
Yep, I know :-) - but it's not needed anymore in any case - the wasm build goes through without it.